doc/source/examples/03_modeling/revolving.mystnb

```diff
@@ -0,0 +1,126 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Modeling: Revolving a sketch
+
+This example shows how use the ``revolve_sketch()`` method to
+revolve a sketch around an axis to create a 3D body. Users can also
+specify the angle of revolution to create a partial body.
+
+```{code-cell} ipython3
+# Imports
+from ansys.geometry.core import Modeler
+from ansys.geometry.core.math import (
+  Plane,
+  Point2D,
+  Point3D,
+  UNITVECTOR3D_X,
+  UNITVECTOR3D_Z,
+)
+from ansys.geometry.core.misc import UNITS, Angle
+from ansys.geometry.core.sketch import Sketch
+
+```
+
+```{code-cell} ipython3
+# Initialize the modeler for this example notebook
+m = Modeler()
+```
+
+## Example: Creating a quarter of a donut
+
+The following code snippets show how to use the `revolve_sketch()` function to create a
```

Comment:
```suggestion
The following code snippets show how to use the ``revolve_sketch()`` function to create a
```
Single backticks render as italics in the doc. You must use double backticks for code entities.